### PR TITLE
Rename field to podRecords

### LINKED ISSUE #521: Rename status field
Run the command `kubectl get timechaos -o yaml`, I found that the status field name is `podChaos`. This is not appropriate here. Maybe it is better to be `podRecord` to be consistent with the new chaos server API.

```
  status:
    experiment:
      endTime: "2020-05-18T18:42:18Z"
      phase: Running
      podChaos:
      - action: ""
        hostIP: 172.17.0.2
        message: time is shifted with -1h
        name: etcd-69bc969869-hjqbm
        namespace: chaos-testing
        podIP: 10.244.1.9
```

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### What problem does this PR solve? 

Fixes #521

### CODE REVIEW

File: api/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -282,6 +282,14 @@ func (in *IoChaosSpec) DeepCopyInto(out *IoChaosSpec) {
...
 		copy(*out, *in)
 	}
+	if in.NextStart != nil {
+		in, out := &in.NextStart, &out.NextStart
+		*out = (*in).DeepCopy()
+	}
+	if in.NextRecover != nil {
+		in, out := &in.NextRecover, &out.NextRecover
+		*out = (*in).DeepCopy()
+	}
```
Comment (Gallardot) [inline api/v1alpha1/zz_generated.deepcopy.go:292]: Maybe you need to update the branch. We made some updates . 
the `nextStart` and `nextRecover` have moved to `status`

#514

Comment (yeya24) [reply api/v1alpha1/zz_generated.deepcopy.go]: Done.

**Pull request merged**